### DISCUSSION TITLE
install之后找不到bypy指令

### DISCUSSION THREAD

**Discussion by czc2000** [houtianze/bypy#656] [Q&A]
install之后找不到bypy指令，输出为`
bypy

Command 'bypy' not found, did you mean:

  command 'pypy' from snap pypy (7.3.12)
  command 'pypy' from deb pypy (7.3.1+dfsg-2)
  command 'mypy' from deb mypy (0.761-1build1)

See 'snap info <snapname>' for additional versions.`

**Comment by lyfllrr** [comment]
加上python命令，
`python -m bypy`或者`python3 -m bypy`
应该就可以了
